doc/modules/changes/20240614_dannberg

```diff
@@ -0,0 +1,7 @@
+Changed: The operator splitting scheme can now use SUNDIALs
+to compute the reactions (rather than a hand-written
+forward Euler scheme). ARKode is the new default, but the
```

Comment:
Can you rephrase this a bit to make clear that ARKode is a part of SUNDIALs?

Reply:
Better?